build/commands/scripts/commands.js

```diff
@@ -58,6 +58,7 @@ program
   .option('--target_os <target_os>', 'target OS')
   .option('--target_arch <target_arch>', 'target architecture')
   .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)', 'classic')
```

Comment:
should this param name change or at least update the description so we know it applies to apk and aab?